.github/workflows/zips.yml

```diff
@@ -113,59 +116,98 @@ jobs:
           name: zip-files
           path: ${{ github.ref }}
 
-  deploy-to-wiki:
-    name: Deploy To Wiki
+  upload-to-cloud-storage:
+    name: Upload to GCS
     runs-on: ubuntu-latest
     needs: build-zips
     steps:
-      - uses: actions/checkout@v3
-        with:
-          repository: ${{ github.repository }}.wiki
-          # GitHub Wikis use this term as their main branch; this cannot be
-          # changed so we need to continue using it for now.
-          ref: master
-      - name: Download artifacts
-        uses: actions/download-artifact@v3
+      - name: Set branch environment variables
+        id: branch-target
+        if: github.event_name != 'pull_request'
+        run: echo "dir=${GITHUB_REF##*/}" >> $GITHUB_OUTPUT
+      - name: Set PR environment variables
+        id: pr-target
+        if: github.event_name == 'pull_request'
+        run: echo "dir=pull/${{ github.event.number }}" >> $GITHUB_OUTPUT
```

Comment:
This doesn't seem to be necessary, at least for the upload. We should be able to get the  path from the `ref` without parsing. It's ok that this adds the `refs` prefix as a top-level folder.

Reply:
This is needed because sometimes `ref` is not just `pull/{issue_id}`, sometimes it is `pull/{issue_id}/merge`. I briefly tried to figure out why it becomes it, but didn't figure it out.

Reply:
That's odd because the wiki only has zips in `refs/pull/{#}/merge/` 🤔 

I'm guessing that the `build-zips` job doesn't run for these events?

Reply:
Ok, updated to use `${{ github.ref }}` as upload paths.